### PR TITLE
Feature/landindingpage2

### PR DESCRIPTION
**Pull request by stevedoty**

### CODE REVIEW

File: public/oldIndex.html [hunk 1]
```diff
@@ -224,8 +224,13 @@ <h2>Stop waiting.<br>Start Getting Involved.</h2>
...
         </p>
         <ul class="list-inline list-social">
+            <li class="list-inline-item ">
+                <a href="https://www.facebook.com/Helping-Families-in-Silicon-Valley-102948694690106?helping=families">
+                    <i class="fab fa-facebook-f"></i>
+                </a>
+            </li>
             <li class="list-inline-item social-instagram">
-                <a href="https://instagram.com/helpingfamiliesinsv">
+                <a href="https://instagram.com/helpingfamiliesinsiliconvalley?helping=families">
```
Comment (montoya332) [inline public/oldIndex.html]: lets add new links to react components . I think there is an object for this

File: public/oldIndex.html [hunk 2]
```diff
@@ -224,8 +224,13 @@ <h2>Stop waiting.<br>Start Getting Involved.</h2>
             Email: <a href="mailto:info@helpingfamiliesinsv.org">info@helpingfamiliesinsv.org</a>
         </p>
         <ul class="list-inline list-social">
+            <li class="list-inline-item ">
+                <a href="https://www.facebook.com/Helping-Families-in-Silicon-Valley-102948694690106?helping=families">
```
Comment (montoya332) [inline public/oldIndex.html]: looks like we added a facebook social.

File: public/oldIndex.html (hunk 1 above)
Comment (stevedoty) [reply public/oldIndex.html]: ok

File: public/oldIndex.html [hunk 3]
```diff
@@ -90,7 +90,7 @@ <h3 class="mb-0">Helpful Tips During Social Distancing</h3>
                         <strong class="d-inline-block mb-2 text-primary">Workshop</strong>
                         <h3 class="mb-0">SMART GOALS</h3>
                         <div class="mb-1 text-muted">March 11</div>
-                        <p class="card-text mb-auto">This workshop provides an overview of a simple but effective model ...</p>
+                        <p class="card-text mb-auto">This workshop provides an overview of a simple but effective model for identifying, planning for business, personal, ...</p>
```
Comment (montoya332) [inline public/oldIndex.html]: were can we add this to view on react app

File: public/oldIndex.html [hunk 4]
```diff
@@ -80,7 +80,7 @@ <h3 class="mb-0">Helpful Tips During Social Distancing</h3>
                         <a href="https://www.eventbrite.com/e/helpful-tips-during-social-distancing-tickets-100336011712" class="stretched-link text-primary">Continue reading</a>
                     </div>
                     <div class="col-auto d-none d-lg-block">
-                        <img class="bd-placeholder-img mt-4" style="max-width: 200px;max-height: 200px;" src=""/>
+                        <img class="bd-placeholder-img mt-4 mr-3" style="max-width: 150px;max-height: 200px;" src="https://img.evbuc.com/https%3A%2F%2Fcdn.evbuc.com%2Fimages%2F96951240%2F408932228207%2F1%2Foriginal.20200319-034420?w=1080&auto=format%2Ccompress&q=75&sharp=10&rect=0%2C0%2C3300%2C1650&s=8bdb790f4499d97a5c49dd3668119e88"/>
```
Comment (montoya332) [inline public/oldIndex.html]: can you update the react section of this. I think adding the source url should be fine

File: src/index.js [hunk 5]
```diff
@@ -1,7 +1,8 @@
 import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
```
Comment (montoya332) [inline src/index.js]: Lets also add the jquery code. not sure if it will work like we expect it though

File: public/oldIndex.html (hunk 1 above)
Comment (stevedoty) [reply public/oldIndex.html]: done

File: public/oldIndex.html (hunk 2 above)
Comment (stevedoty) [reply public/oldIndex.html]: done

File: public/oldIndex.html (hunk 3 above)
Comment (stevedoty) [reply public/oldIndex.html]: done

**Pull request merged**